pkg/scheduler/framework/session_plugins.go

```diff
@@ -304,6 +309,50 @@ func (ssn *Session) JobOrderFn(l, r interface{}) bool {
 
 }
 
+// NamespaceOrderEnabled returns the NamespaceOrder is enabled in this session or not
+// judging by length of namespaceOrderFns
+func (ssn *Session) NamespaceOrderEnabled() bool {
+	return len(ssn.namespaceOrderFns) > 0
+}
+
+// NamespaceOrderEnabledInPlugin the NamespaceOrder for this plugin is enabled in this session or not
+func (ssn *Session) NamespaceOrderEnabledInPlugin(pluginName string) bool {
```

Comment:
move this to drf plugin; otherwise, there're lots of helper function in `Session`.

Reply:
move it to `drf.NamespaceOrderEnabled`.